packages/react-native-web/src/exports/Touchable/index.js

```diff
@@ -73,6 +72,11 @@ const extractSingleTouch = (nativeEvent) => {
  *     return merge(this.touchableGetInitialState(), yourComponentState);
  *   }
  *
+ * - Add a method to get your touchable component's node.
+ *   getTouchableNode: function() {
+ *     return this.touchableRef.current
+ *   }
```

Comment:
Where is this used / implemented?

Reply:
Guidance from React is to remove `findDOMNode` and use React.Ref instead.  The `Touchable` and `ScrollResponder` mixins don't render themselves, so they become dependent on the concrete component to capture the root to a ref and provide it.  

I didn't see any uses of `Touchable` directly in this repository, so that seems to be only a user-facing breaking change.  Nothing in this repository actually implements it.

`ScrollResponderMixin` was used by `ScrollView`, which already provided `getScrollableNode`.  It appears that it would only be user components using `ScrollResponderMixin` that would have to be sure to implement `getScrollableNode` now.

Reply:
I think after this is merged, it will also be a good time to remove those mixins as a legacy feature that will not carry forward with React 18 adoption.